net/nmap/Makefile

```diff
@@ -76,6 +76,13 @@ $(call Package/nmap/default)
   TITLE:=Network packet generation tool / ping utility
 endef
 
+define Package/nping-ssl
+$(call Package/nmap/default)
+  DEPENDS:=$(NPING_DEPENDS) +libopenssl
+  VARIANT:=ssl
+  TITLE:=Network packet generation tool / ping utility (with OpenSSL support)
```

Comment:
This title is really long and that's why it isnt shown in `make menuconfig`.
![image](https://user-images.githubusercontent.com/4096468/65234366-b7052380-dad4-11e9-881f-ef4324a2bbd2.png)